mkdocs.yml

```
@@ -111,6 +111,7 @@ nav:
           - Upgrading Nautobot: "installation/upgrading.md"
           - Upgrading Nautobot from v1.X to v2.0: "installation/upgrading-from-nautobot-v1.md"
           - Migrating from PostgreSQL to MySQL: "installation/migrating-from-postgresql.md"
+          - Migrating from Region and Site to Location: "installation/region-and-site-data-migration-guide.md"
```

Perhaps a more intuitive ordering?

```suggestion
          - Migrating from PostgreSQL to MySQL: "installation/migrating-from-postgresql.md"
          - Upgrading Nautobot: "installation/upgrading.md"
          - Upgrading Nautobot from v1.X to v2.0: "installation/upgrading-from-nautobot-v1.md"
          - Migrating from Region and Site to Location: "installation/region-and-site-data-migration-guide.md"
```